source/_docs/z-wave/device-specific.markdown

```diff
@@ -68,37 +70,10 @@ Reboot your Pi 4 without the Razberry Z-Wave hat first. Then shutdown, add the h
 #### Raspberry Pi 3 procedure
 
 ```text
-dtoverlay=pi3-disable-bt
-```
-
-Then disable the Bluetooth modem service:
-
-```bash
-sudo systemctl disable hciuart
-```
-
-Once Bluetooth is off, enable the serial interface via the `raspi-config` tool. After reboot run:
-
-```bash
-sudo systemctl mask serial-getty@ttyAMA0.service
-```
-
-so that your serial interface looks like:
-
-```text
-crw-rw---- 1 root dialout 204, 64 Sep  2 14:38 /dev/ttyAMA0
-```
-at this point simply add your user (homeassistant) to the dialout group:
-
```

Comment:
This might still be correct, depending on the installation method one is using.

Reply:
I think having all of these comments (which are very hard to actually do on Home Assistant OS) is more confusing than helpful. I struggled alot to figure out what to actually do to enable support in Home Assistant OS as there is so much outdated, wrong and unnescesarry info around and the corret steps for Home Assistant OS was nowhere to be found.

Reply:
Yes, I agree with you.
However just deleting things, just because they don't apply to you, isn't helping others that do need this.

Hence, in my above comment, it might be wise to add different sections for specific installation types.

Reply:
I agree that I might have hit that delete button a bit hard. I have added the steps I know works for Home Assistant OS, and I have tried to the best of my abilites looked into what should be needed to enable in a more general matter. I would still like to delete most of these commands as they seems outdated and would probalby cause more harm than good in this document. I base this on that they were commited in 2018 ( https://github.com/home-assistant/home-assistant.io/pull/6162 ) and actually seems to target Home Assistant OS (user homeassistant), and I know throught my experience that these commands are no longer needed on Home Assistant OS.